build.gradle

```diff
@@ -16,9 +16,6 @@ repositories {
   mavenCentral()
 }
 
-if (JavaVersion.current() != JavaVersion.VERSION_1_8) {
-  throw new GradleException("This build must be run with Java 8")
-}
```

Comment:
do we need this? I have java 9

Reply:
Actually, we can remove. This check was more for guarding against compatibility changes and ensure we build with a stable version (but java 9 is fairly stable).

Reply:
Should we be using Java 9 with this project going forward?

Reply:
Sure. Java 9 is fairly stable anyways. Also comes with convenient factory methods for collections!